task.js

```diff
@@ -0,0 +1,64 @@
+window.onload = function() {
```

Comment:
アロー関数を使った方が良いかと思いました。

```suggestion
window.onload = () => {
```

Reply:
アロー関数にしました。